src/cosalib/aws.py

```diff
@@ -33,7 +33,7 @@ def delete_snapshot(snap_id, region):
 def aws_run_ore_replicate(build, args):
     build.refresh_meta()
     buildmeta = build.meta
-    if len(buildmeta['amis']) < 1:
+    if 'amis' not in buildmeta:
```

Comment:
Hmm, I think we do want to check that there's at least one source AMI though, no? I mean, we'll error out anyway later on at:

```python
        args.source_region = buildmeta['amis'][0]['name']
```

But the error message here is nicer than what `IndexError` will print. :)   So maybe just:

```python
if len(buildmeta.get('amis', [])) < 1:
```

?